lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -305,3 +307,22 @@ multiSettlementHandler :: [CreditRecord] -> LndrHandler NoContent
 multiSettlementHandler transactions = do
     result <- mapM (uncurry submitHandler) $ zip [0..(length transactions)] transactions
     return $ head result
+
+
+requestPayPalHandler :: PayPalRequest -> LndrHandler NoContent
+requestPayPalHandler (PayPalRequest friend requestor _) = do
+    (ServerState pool configTVar loggerSet) <- ask
+    config <- liftIO $ readTVarIO configTVar
+
+    let attemptToNotify = do
+            pushDataM <- liftIO . withResource pool $ Db.lookupPushDatumByAddress friend
+            nicknameM <- liftIO . withResource pool $ Db.lookupNick requestor
+
+            forM_ pushDataM $ \(channelID, platform) -> liftIO $ do
+                responseCode <- sendNotification config (Notification channelID platform nicknameM RequestPayPal )
+                let logMsg = "Notification response (PayPal Request): " ++ show responseCode
+                pushLogStrLn loggerSet . toLogStr $ logMsg
+
+    attemptToNotify
+    
+    pure NoContent
```

Comment:
Missing an end-of-line. (sorry to make these comments about whitespace, it has to be fixed at some point...)